pulp_rpm/app/models/repository.py

```diff
@@ -51,6 +57,9 @@ class RpmRepository(Repository):
     ]
 
     sub_repo = models.BooleanField(default=False)
+    revision_number = models.IntegerField(null=True)
+    remote = models.ForeignKey(Remote, null=True, on_delete=models.SET_NULL)
+    last_sync_repo_version = models.PositiveIntegerField(default=0)
```

Comment:
@goosemania What do you think about changing the names of these to last_sync_revision_number and last_sync_remote? 

I don't feel strongly about it.

Reply:
No strong preference either.

I would agree that the proposed names are more consistent and descriptive, at the same time they can only refer to the last sync. Maybe less assumptions is better and they are internal anyway and should not be exposed  in REST API, so length doesn't really matter.

Long story short, +1 to your suggestion :)